Python helper scrips for Politehnica University of Timisoara Big Data in Health and Bioinformatics laboratory

**Bibliography: **
**Textbooks**

Barabási, A.-L. (2016). Network Science. Cambridge University Press.

Available online: https://barabasi.com/networksciencebook/

Bishop, C. M. (2006). Pattern Recognition and Machine Learning. Springer.

Brown, S. M. (2015). Next-Generation DNA Sequencing Informatics (2nd ed.). Cold Spring Harbor Laboratory Press.

Géron, A. (2019). Hands-On Machine Learning with Scikit-Learn, Keras, and TensorFlow (2nd ed.). O'Reilly Media.

Gelman, A., Carlin, J. B., Stern, H. S., Dunson, D. B., Vehtari, A., & Rubin, D. B. (2013). Bayesian Data Analysis (3rd ed.). CRC Press.

Lesk, A. M. (2019). Introduction to Bioinformatics (5th ed.). Oxford University Press.

Newman, M. E. J. (2010). Networks: An Introduction. Oxford University Press.

Sivia, D. S., & Skilling, J. (2006). Data Analysis: A Bayesian Tutorial (2nd ed.). Oxford University Press.

**Articles and Papers**

Altschul, S. F., Gish, W., Miller, W., Myers, E. W., & Lipman, D. J. (1990). Basic local alignment search tool. Journal of Molecular Biology, 215(3), 403-410.

Andrews, S. (2010). FastQC: A Quality Control Tool for High Throughput Sequence Data. Babraham Bioinformatics.

Available online: https://www.bioinformatics.babraham.ac.uk/projects/fastqc/

Bolstad, B. M., Irizarry, R. A., Åstrand, M., & Speed, T. P. (2003). A comparison of normalization methods for high density oligonucleotide array data based on variance and bias. Bioinformatics, 19(2), 185-193.

Cheng, F., Zhou, Y., Li, J., Li, W., Liu, G., & Tang, Y. (2012). Prediction of chemical-protein interactions network with weighted network-based inference method. PLoS One, 7(7), e41064.

Cook, C. E., Bergman, M. T., Finn, R. D., Cochrane, G., Birney, E., & Apweiler, R. (2016). The European Bioinformatics Institute in 2016: Data growth and integration. Nucleic Acids Research, 44(D1), D20-D26.

Eisen, M. B., Spellman, P. T., Brown, P. O., & Botstein, D. (1998). Cluster analysis and display of genome-wide expression patterns. Proceedings of the National Academy of Sciences, 95(25), 14863-14868.

Jeong, H., Mason, S. P., Barabási, A.-L., & Oltvai, Z. N. (2001). Lethality and centrality in protein networks. Nature, 411(6833), 41-42.

Langfelder, P., & Horvath, S. (2008). WGCNA: An R package for weighted correlation network analysis. BMC Bioinformatics, 9, 559.

Li, H., & Durbin, R. (2009). Fast and accurate short read alignment with Burrows-Wheeler transform. Bioinformatics, 25(14), 1754-1760.

Libbrecht, M. W., & Noble, W. S. (2015). Machine learning applications in genetics and genomics. Nature Reviews Genetics, 16(6), 321-332.

Pushpakom, S., Iorio, F., Eyers, P. A., Escott, K. J., Hopper, S., Wells, A., ... & Pirmohamed, M. (2019). Drug repurposing: Progress, challenges and recommendations. Nature Reviews Drug Discovery, 18(1), 41-58.

van de Schoot, R., Kaplan, D., Denissen, J., Asendorpf, J. B., Neyer, F. J., & van Aken, M. A. G. (2014). A gentle introduction to Bayesian analysis: Applications to developmental research. Child Development, 85(3), 842-860.

Vogelstein, B., Papadopoulos, N., Velculescu, V. E., Zhou, S., Diaz, L. A., & Kinzler, K. W. (2013). Cancer genome landscapes. Science, 339(6127), 1546-1558.

Zhang, B., & Horvath, S. (2005). A general framework for weighted gene co-expression network analysis. Statistical Applications in Genetics and Molecular Biology, 4(1), Article17.

**Online Resources**

**Biopython Documentation**

Biopython Tutorial and Cookbook: https://biopython.org/DIST/docs/tutorial/Tutorial.html

**Bioconductor Project**

https://www.bioconductor.org/

**The Cancer Genome Atlas (TCGA)**

https://www.cancer.gov/about-nci/organization/ccg/research/structural-genomics/tcga

**Cytoscape Tutorials**

https://cytoscape.org/learn.html

**NCBI Education Resources**

https://www.ncbi.nlm.nih.gov/home/learn/

**FastQC Software and Documentation**

https://www.bioinformatics.babraham.ac.uk/projects/fastqc/

**Genomic Data Commons (GDC) API**

https://gdc.cancer.gov/developers/gdc-application-programming-interface-api

**NetworkX Documentation**

https://networkx.org/documentation/

**Notes:**

Biopython: A set of freely available tools for biological computation written in Python by an international team of developers. The tutorial and cookbook provide extensive examples on how to use Biopython for bioinformatics tasks.

Bioconductor: An open-source project providing tools for the analysis and comprehension of high-throughput genomic data in R.

FastQC: A quality control tool for high-throughput sequence data, providing insights into the quality of sequencing data and identifying any potential problems.

Cytoscape: An open-source software platform for visualizing complex networks and integrating these with any type of attribute data.

NetworkX: A Python package for the creation, manipulation, and study of the structure, dynamics, and functions of complex networks.
